crm_compassion/migrations/14.0.1.2.0/pre-migration.py

```diff
@@ -0,0 +1,9 @@
+def migrate(cr, version):
```

Comment:
doesn't really change much... but shouldn't this be in a `post-migration` since we are not changing the structure of the database here